src/mmw/sass/components/_layerpicker.scss

```diff
@@ -115,13 +111,15 @@
 .layerpicker-legendcolor {
```

Comment:
Since NLCD has a white item and many have faint items, I think this could use a border.  `border: 1px solid #ccc` looked fine to me, but @jfrankl may want something specific.